tools/Experiments/src/DevHome.Experiments.csproj

```diff
@@ -1,20 +1,29 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
-  <Import Project="$(SolutionDir)ToolingVersions.props"/>
+  <Import Project="$(SolutionDir)ToolingVersions.props" />
   <PropertyGroup>
     <RootNamespace>DevHome.Experiments</RootNamespace>
     <Platforms>x86;x64;arm64</Platforms>
     <RuntimeIdentifiers>win10-x86;win10-x64;win10-arm64</RuntimeIdentifiers>
     <UseWinUI>true</UseWinUI>
   </PropertyGroup>
   <ItemGroup>
+    <None Remove="Views\QuietBackgroundProcessesPage.xaml" />
```

Comment:
The QuiteBackgroundProcesses stuff should go in its own tools/QuietBackgroundProcesses project.  All code should be written as if it wasn't an experimental feature, then the experimental feature checks just wrap it to enable/disable it.  That way when it's not experimental anymore, we just remove the checks and the rest of the code stays exactly the same.